Solutions/CohesitySecurity/Tests/az.py

```diff
@@ -79,6 +85,28 @@ def run_playbook(
         ],
         stdout=subprocess.PIPE,
     )
+
+    while True:
+        playbook_run = get_latest_playbook_run(
+            access_token,
+            subscription_id,
+            resource_group,
+            playbook_name,
+        )
+        if playbook_run["value"][0]["properties"]["status"] != "Running":
```

Comment:
This can lead to an infinite loop if the playbook is stuck for any reason. Please set a reasonable timeout (5 min) and crash the program.

Reply:
done